.travis.yml

```diff
@@ -1,29 +1,43 @@
 # Travis CI configuration file
 # http://about.travis-ci.org/docs/
 
+dist: xenial
+
 language: python
 
+
+
 python:
     - "2.7"
     - "3.5"
     - "3.6"
+    - "3.7"
 
 env:
-  - TEST_PARAM="--unit"
-  - TEST_PARAM="--integ"
+  - TEST_PARAM="--unit -v"
+  - TEST_PARAM="--integ -v"
 
 before_install:
   - sudo apt-get update -qq
-  - sudo apt-get install -qq tor
+  - sudo apt-get install -qq libevent-dev libssl-dev zlib1g-dev python-tox python-dev python3-dev wget
+  - cd $HOME
+  - wget https://dist.torproject.org/tor-0.4.0.5.tar.gz
+  - tar zxf tor-0.4.0.5.tar.gz && cd tor-0.4.0.5
+  - ./configure && make
 
 # Dependencies installation commands
 install:
+  - cd $TRAVIS_BUILD_DIR
   - pip install -U pip
   - pip install -r requirements.txt
+  - pip install wrapt_timeout_decorator
 
 # Tests running commands
 script:
-  - ./run_tests.py $TEST_PARAM 
+  - TOR_BINARY=$HOME/tor-0.4.0.5/src/app/tor python run_tests.py -v $TEST_PARAM --tor $HOME/tor-0.4.0.5/src/app/tor
+
+after_failure:
+  - cat /home/travis/build/rj76/stem/test/data/tor_log
```

Comment:
Should this (personalized) path be included in this PR?

Reply:
it just won't run there; I'd `git rm .travis.yml`, but yeah, reset to master (where it won't run either)

Reply:
https://dpaste.de/fysX

builds well on modern ubuntu jenkins